apis/swagger.yml

```diff
@@ -629,10 +629,9 @@ definitions:
         type: "string"
         description: "IP address which peer client carries"
         format: "ipv4"
-      superNodeIp:
+      superNode:
```

Comment:
It's not a good idea to change the parameter's name of a API. It will cause compatibility problems.

Reply:
Done. PTAL again.